Packs/CommonScripts/Scripts/ShowIncidentIndicators/ShowIncidentIndicators.py

```diff
@@ -0,0 +1,42 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+
+def group_by_type(indicators):
+    """
+    Gets the indicators of the current incident.
+
+    Returns:
+        A list of the indicator types followed by the corresponding indicator values.
+    """
+
+    grouped = {}
+    for indicator in indicators:
+        if indicator['indicator_type'] not in grouped:
+            grouped[indicator['indicator_type']] = [indicator['value']]
+        else:
+            grouped[indicator['indicator_type']].append(indicator['value'])
+    result = []
+    for key, values in grouped.items():
```

Comment:
If `key, values` have mentionable roles (indicator value, indicator name, etc..) it is better to use their names for the readability of the code

Reply:
got it